<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Recursos/lista-de-verificacion.png</normaloff>:/Recursos/lista-de-verificacion.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>295</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>259</width>
       <height>301</height>
      </size>
     </property>
     <property name="title">
      <string>Notas</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QTableWidget" name="outNotas"/>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>101</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>18</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Promedio:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="outProm">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>206</width>
       <height>301</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>206</width>
       <height>301</height>
      </size>
     </property>
     <property name="title">
      <string>Ingreso Notas</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inNombre"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nota:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="inNota">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="btnAgregar">
        <property name="text">
         <string>Agregar</string>
        </property>
        <property name="icon">
         <iconset resource="Recursos.qrc">
          <normaloff>:/Recursos/boton-agregar.png</normaloff>:/Recursos/boton-agregar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="btnDibujar">
        <property name="text">
         <string>Dibujar</string>
        </property>
        <property name="icon">
         <iconset resource="Recursos.qrc">
          <normaloff>:/Recursos/barra-grafica.png</normaloff>:/Recursos/barra-grafica.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="outDiagrama">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
